content/blog/2022-05-04-e2e-cv-pt2.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'End-to-End Computer Vision API, Part 2: Local Experiments'
+date: 2022-05-04
+description: >
+  In part 1 of this series of posts, we introduced a solution to a common
+  problem faced by companies in the manufacturing industry: detecting defects
+  from images of products moving along a production line. The solution we
+  proposed was a Deep Learning-based image segmentation model wrapped in a web
+  API. We talked about effective management and versioning of large datasets and
+  the creation of reproducible ML pipelines.
```

Comment:
But what will we do in this part?  You could summarize what was in first part and add what will happen in Part 2.

Reply:
Paraphrased this because this short description gets truncated and you can't see it in full anyway.